server/bin/pbench-unpack-tarballs.py

```diff
@@ -44,13 +42,13 @@ def main(cfg_name):
     init_db(config, logger)
 
     # Initiate the unpacking
-    unpack_obj = Unpack(config, _linksrc, logger)
-    ntotal, nsuccess = unpack_obj.unpack_tarball()
+    unpack_obj = Unpack(config, logger)
+    result = unpack_obj.unpack_tarball()
 
     result_string = (
-        f"Total processed: {ntotal},"
-        f" Total successes: {nsuccess},"
-        f" Total failures: {ntotal-nsuccess}"
+        f"Total processed: {result.ntotal},"
+        f" Total successes: {result.nsuccess},"
+        f" Total failures: {result.ntotal-result.nsuccess}"
```

Comment:
Fascinating!  We've found a case which Black doesn't care about!!  Apparently, it ignores expressions inside the braces in f-strings.

Could you please add spaces on either side of the minus-sign, since that's what Black would have done if it expression weren't embedded?  🙂